### DISCUSSION TITLE
How to find the webui API

### DISCUSSION THREAD

**Discussion by jynrock** [open-webui/open-webui#1179] [General]
Hello,
I'm trying to connect an android chatbot app to my server hosting Ollama but I always get a connection error.
I'm using Ollama through OpenWeb-ui on Docker.
I have tried with Ollama Base URL :
http://host.docker.internal:11434
Or by replacing "host.docker.internal" with the local ip of my server.
I always get a connection error.

**Comment by layogtima** [comment]
Same! I'm using ngrok to tunnel through, and I tried a bunch of things, but no dice yet!

**Comment by ghost** [comment]
@jynrock @layogtima What chatbot do you use?

**Comment by yotaken** [comment]
> Hello, I'm trying to connect an android chatbot app to my server hosting Ollama but I always get a connection error. I'm using Ollama through OpenWeb-ui on Docker. I have tried with Ollama Base URL : http://host.docker.internal:11434 Or by replacing "host.docker.internal" with the local ip of my server. I always get a connection error.

 When I encounter connection issues between Docker containers of different projects, what I usually do is run "docker network inspect" to know which network is using my service and see which gateway the container uses to go outside of the network.
 
Example to clarify:

```
docker network ls

NETWORK ID     NAME                 DRIVER    SCOPE
uuid   app_container    bridge    local

docker network inspect app_container
[...]
 "IPAM": {
            "Driver": "default",
            "Options": null,
            "Config": [
                {
                    "Subnet": "[172.20.0.0/16](http://172.20.0.0/16)",
                    "Gateway": "172.20.0.1"
                }
            ]
        },
```

So the url should be in this example: http://172.20.0.1:11434
